README.rst

```diff
@@ -49,3 +49,21 @@ Repository Overview
 .. _pypyjs-examples: https://github.com/pypyjs/pypyjs-examples
 .. _pypyjs.github.io: https://github.com/pypyjs/pypyjs.github.io
 .. _pypyjs.org: https://pypyjs.org
+
+Transpiling es6
+~~~~~~~~~~~~~~~
+
+Pypyjs is written using es6 and is transpiled to es5 using `babeljs`_ to run the
+code though the transpiler you need `gulp`_ and you'll need a few other
+node_modules, and nodejs ofcourse (I'm assuming you have those installed, npm
+comes with node).
+
+To install the node_modules you need to run `npm install` in the root of the
+repository. And you'll need to install gulp globaly like this
+`npm install gulp -g`.
+
+When you run `gulp` in the repository the es6 source in /src is transpiled to
+es5 and thats copied to /lib.
```

Comment:
Should we also add this to the various recipes in the Makefile, to ensure it gets run when building releases etc?